test/sanity/code-smell/mypy.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I think you may want to add `prefixes` here (see `obsolete-files.json`) so this test only runs if files in the specified directories are changed

Reply:
@gundalow it's best to run mypy against whole packages rather than individual modules so I'm not sure. I'll look at that example tho.

Reply:
Is it documented somewhere how this works?

cc @mattclay

Reply:
The existing sanity tests provide examples for the prefixes. There isn't any documentation.

Whether we use prefixes or not will depend on how mypy behaves when given individual files vs modules.